dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.httpStat-monitor1.0.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
Should you name the feature `io.openliberty.httpStat-monitor-1.0.feature`?

Reply:
Will rename both to:
`io.openliberty.http.monitor-1.0`
`io.openliberty.http.monitor-1.0.jakarta.feature`

Reply:
The existing patterns usually follow like
`io.openliberty.<featureA>-<featureB>`
so something like
`io.openliberty.servlet3.1-monitor1.0` and `io.openliberty.servlet5.0-monitor1.0`

https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-naming-guidelines#features

Reply:
@tevans78 Even if the auto feature targets starting http monitor bundle? (Using servlets since the core of it relies on the servlet filter to function properly)